Composer/packages/lib/shared/src/featureFlagUtils/Feature.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import React, { Fragment } from 'react';
+
+import { FeatureFlagMap, FeatureFlagNames, getFeatureFlagValue } from './DefaultFeatureFlags';
+
+type FeatureProps = {
+  featureFlagName: FeatureFlagNames;
+  featureFlagMap: FeatureFlagMap;
```

Comment:
I don't think this should be passed to this component, this component needs to be "smart" and know about the map within.